dist/components/OutboundLink.js

```diff
@@ -0,0 +1,34 @@
+var React = require('react');
+var assign = require('react/lib/Object.assign');
+
+var OutboundLink = React.createClass({
+  displayName: 'OutboundLink',
+  statics: {
+    trackLink: function() {
+      console.warn("ga tracking not enabled");
+    }
+  },
+  handleClick: function (e) {
+    e.preventDefault();
+    var props = this.props;
+    OutboundLink.trackLink(props.args, function () {
+      if ( props.target === '_blank' ) {
+        window.open(props.to, '_blank');
```

Comment:
nit: string repetition, it's probably better to cache that string somewhere up top and then use `if ( props.target === NEWTAB ) { window.open(props.to, NEWTAB); }` or just use `window.open(props.to, props.target);` since we already know that it's the string "_blank" here.